crates/runtime/src/modelsource.rs

```diff
@@ -2,6 +2,7 @@ use crate::auth::AuthProvider;
 use snafu::prelude::*;
 use std::collections::HashMap;
 use std::sync::Arc;
+use tonic::async_trait;
```

Comment:
It's weird that we're using the tonic async_trait here. tonic is a GRPC library - what does that have to do with a model source?

Reply:
Updated to import async-trait explicitly from async-trait https://github.com/spiceai/spiceai/pull/707/commits/eb0d8680c761edaeb072a4c888ac44fa0712fcf9

Previously:
tonic::async_trait was  a re-export of [async-trait](https://docs.rs/async-trait) (see https://docs.rs/tonic/latest/tonic/attr.async_trait.html)

async-trait is recommended solution by rust async book: https://rust-lang.github.io/async-book/07_workarounds/05_async_in_traits.html